hub/constraints/instruction-handling/copy/extcodecopy.lisp

```diff
@@ -18,24 +18,28 @@
                                     (* (copy-instruction---foreign-address-warmth)       GAS_CONST_G_WARM_ACCESS)
                                     (* (- 1 (copy-instruction---foreign-address-warmth)) GAS_CONST_G_COLD_ACCOUNT_ACCESS)))))
 
-(defconstraint copy-instruction---EXTCODECOPY---the-MXPX-case (:guard (copy-instruction---standard-EXTCODECOPY))
-               (if-not-zero stack/MXPX
-                            (execution-provides-empty-return-data ROFF_EXTCODECOPY_MXPX_CONTEXT_ROW)))
+;; ;; completely redundant constraint:
+;; (defconstraint copy-instruction---EXTCODECOPY---the-MXPX-case (:guard (copy-instruction---standard-EXTCODECOPY))
+;;                (begin
+;;                  (debug
+;;                    (if-not-zero stack/MXPX
+;;                                 (execution-provides-empty-return-data    ROFF_EXTCODECOPY_MXPX_CONTEXT_ROW)))))
```

Comment:
This is redundant: whenever `XAHOY = 1` we finish on `(execution-provides-empty-return-data ...)`